$MirOS: ports/lang/python/2.3/patches/patch-Lib_test_regrtest_py,v 1.2 2005/11/15 19:16:58 tg Exp $
$OpenBSD: patch-Lib_test_regrtest_py,v 1.1.1.1 2003/12/31 17:38:33 sturm Exp $
--- Lib/test/regrtest.py.orig	2007-03-08 20:58:46.000000000 +0100
+++ Lib/test/regrtest.py	2008-04-22 23:06:09.000000000 +0200
@@ -1310,6 +1310,31 @@ _expectations = {
         test_winreg
         test_winsound
         """,
+     'openbsd3':
+         """
+       test_aepack
+       test_al
+       test_cd
+       test_cl
+       test_dl
+       test_email_codecs
+       test_gl
+       test_imgfile
+       test_linuxaudiodev
+       test_locale
+       test_macfs
+       test_macostools
+       test_nis
+       test_normalization
+       test_ossaudiodev
+       test_pep277
+       test_plistlib
+       test_scriptpackages
+       test_sunaudiodev
+       test_unicode_file
+       test_winreg
+       test_winsound
+        """,
 }
 _expectations['freebsd5'] = _expectations['freebsd4']
 _expectations['freebsd6'] = _expectations['freebsd4']
